### ISSUE TITLE
Repeating element with minOccurs="0" and maxOccurs="unbounded" still marked as required

### ISSUE THREAD

**Issue by Tomas-Kraus** [eclipse-ee4j/jaxb-ri#210]
Hello,

The following schema:

<xs:element name="MobProduct">
<xs:complexType>
<xs:sequence>
<xs:element ref="MobOption" minOccurs="0" maxOccurs="unbounded" />
<xs:element ref="MobService" minOccurs="0" maxOccurs="unbounded" />
<xs:element ref="MobBundle" minOccurs="0"/>
<xs:element ref="MNP" minOccurs="0" maxOccurs="unbounded" />
</xs:sequence>
</xs:complexType>
</xs:element>

generates
...
@XmlRootElement(name = "MobProduct")
public class MobProduct
implements Serializable
{

@XmlElement(name = "MobOption", required = true)
protected List<MobOption> mobOption;
@XmlElement(name = "MobService", required = true)
protected List<MobService> mobService;
@XmlElement(name = "MobBundle")
protected MobBundle mobBundle;
@XmlElement(name = "MNP", required = true)
protected List<MNP> mnp;

In other words, when minOccurs="0" and maxOccurs="unbounded", the elements are
incorrectly marked as required.

Thanks,
Axel
#### Environment
Operating System: All
Platform: All
#### Affected Versions
[2.0.1]
